docs/src/main/paradox/index.md

```diff
@@ -1,273 +1,93 @@
-
-![Frontend to Master Message Flow](images/frontend-master-message-flow.svg)
-
-When a worker receives <code>WorkIsReady</code> it sends back <code>WorkerRequestsWork</code>
-to the master, which hands out the work, if any, to the worker. The master keeps track of that
-the worker is busy and expect a result within a deadline. For long running jobs the worker
-could send progress messages, but that is not implemented in the example.
-
-![Master to Worker Message Flow](images/master-worker-message-flow.svg)
-
-When the worker sends <code>WorkIsDone</code> the master updates its state of the worker
-and sends acknowledgement back to the worker. This message must also be idempotent as the worker will
-re-send if it doesn't receive the acknowledgement.
-
-![When Work is Done](images/master-worker-message-flow-2.svg)
-
-## Summary
-
-The <a href="#code/src/main/scala/worker/Master.scala" class="shortcut">Master</a> actor
-is a <a href="http://doc.akka.io/docs/akka/2.4.0/scala/cluster-singleton.html"
-target="_blank">Cluster Singleton</a> and register itself in the 
-<a href="http://doc.akka.io/docs/akka/2.4.0/scala/cluster-client.html" target="_blank">Cluster Receptionist</a>.
-The <code>Master</code> is using 
-<a href="http://doc.akka.io/docs/akka/2.4.0/scala/persistence.html" target="_blank">Akka Persistence</a>
-to store incoming jobs and state.
-
-The <a href="#code/src/main/scala/worker/Frontend.scala" class="shortcut">Frontend</a> actor send work
-to the master via the <code>ClusterSingletonProxy</code>.
-
-The <a href="#code/src/main/scala/worker/Worker.scala" class="shortcut">Worker</a> communicate with the
-cluster and its master with the <a href="http://doc.akka.io/docs/akka/2.4.0/scala/cluster-client.html"
-target="_blank">Cluster Client</a>.
-
-![Cluster Nodes](images/cluster-nodes.svg)
-
-## Run the Application
-
-Open the <a href="#run" class="shortcut">Run</a> tab and select <code>worker.Main</code> followed
-by Restart. On the left-hand side we can see the console output, which is logging output
-from nodes joining the cluster, the simulated work and results.
-
-The <a href="#code/src/main/scala/worker/Main.scala" class="shortcut">worker.Main</a>
-starts three actor systems in the same JVM process. It can be more
-interesting to run them in separate processes. <b>Stop</b> the application in the
-<a href="#run" class="shortcut">Run</a> tab and then open three terminal windows.
-
-In the first terminal window, start the first seed node with the following command:
-
-```bash
-sbt "runMain worker.Main 2551"
+1. Download the zip file from [Lightbend Tech Hub](http://dev.lightbend.com/start/?group=akka&project=akka-distributed-workers-scala) by clicking `CREATE A PROJECT FOR ME`. 
+1. Extract the zip file to a convenient location: 
+  - On Linux and OSX systems, open a terminal and use the command `unzip akka-distributed-workers-scala.zip`. Note: On OSX, if you unzip using Archiver, you also have to make the sbt files executable:
```

Comment:
Apparently, they are calling the more recent versions macOS (instead of OSX), so maybe you want to replace it in the note?